test/ui/spec/defaultHtmlSequencerUi.spec.js

```diff
@@ -4,6 +4,10 @@ describe('Default sequencer HTML', function() {
   var sequencer = require('../../../src/ImageSequencer')();
```

Comment:
So, here it looks like this is already done in the default UI code. So maybe we need to split this into another file to have one running purely on the fixture file.